service/dap/stdio_pipe.go

```diff
@@ -0,0 +1,13 @@
+//go:build !linux
+// +build !linux
+
+package dap
+
+import (
+	"errors"
+)
+
+func generateStdioTempPipes() (res [2]string, err error) {
+	err = errors.New("Unimplemented")
```

Comment:
Merging this with just a linux implementation would be weird. This is enough of a prominent feature that having it implemented for one operating system implies that we have to implement it for at least the more important ones. I think it needs to support Windows and macOS.

Reply:
Implementing it to all platforms should be relatively easier if we can change the `Redirects` to not only be paths to `stdout`/`stderr`, but `io.Writer`s. I'll implement complete support for `native`, which should be relatively easy to do soin that backend.